docs/pages/inboxes/manage-inboxes.mdx

```diff
@@ -36,6 +36,16 @@ This function is delicate and should be used with caution. Adding an identity to
 
 :::code-group
 
+```tsx [Browser]
+// where identityToAdd is a signer for the identity to add
+await client.unsafe_addAccount(identityToAdd, true)
+```
+
+```tsx [Node]
+// where identityToAdd is a signer for the identity to add
+await client.unsafe_addAccount(identityToAdd, true)
+```
+
```

Comment:
@rygine Did I interpret this feedback correctly for the Browser and Node examples above? Or should `identityToAdd` be changed to `signer`? =)

>these methods accept a signer as the first argument, and must pass true as the 2nd argument.

Reply:
should be changed to `signer`